packages/perseus/src/styles/widgets/passage.less

```diff
@@ -46,11 +46,15 @@
         margin: 0 0 10px;
     }
     .passage-text div.paragraph {
-        font-family: Lato, "Noto Sans", Helvetica, Corbel, sans-serif, Helvetica, Corbel, sans-serif;
+        font-family: "minion-pro", KaTeX_Main, Times, "Times New Roman", serif;
         font-weight: 400;
-        font-size: 14px;
-        line-height: 16px;
+        font-size: @font-size;
+        line-height: @line-height;
         text-indent: 20px;
+        // note(matthewc): If you need to change this,
+        // check that line numbers line up correctly.
+        // The margin should probably be 0 or the line height
```

Comment:
It might be worth pulling this comment up to the top of this rule and noting that any changes could cause line numbering to break.

Reply:
Done and done: https://khanacademy.atlassian.net/browse/LP-13275